src/app/components/Contributor.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ContributorProps } from "@/types";
+
+const Contributor = ({ contributors }: ContributorProps) => {
+  const formattedContributors = contributors.map((contributor, index) => {
+    const name = `${contributor.first_name} ${contributor.last_name[0]}`;
+
+    return (
```

Comment:
Bro please return it as a string like "Anuj M, Auguestine M, ...." and not as a component